src/plugins/github/demoData/README.md

```diff
@@ -0,0 +1,18 @@
+# DemoData Instructions
+
+This folder contains example data for testing the GitHub plugin. The example
+data is generated by running our GitHub API fetch against the SourceCred
+example repo at https://github.com/sourcecred/example-repo/.
+
+The demo data should be regenerated any time either the example repository or
+the GitHub api query change. To regenerate the demo data, run the following
+command:
+
+`
+$ GITHUB_TOKEN="your_token_here"; src/plugins/github/demoData/regenerate.sh;
```

Comment:
Can we use code fences here (and below)?
``````markdown
```shell
$ GITHUB_TOKEN="your_token_here" src/plugins/github/demoData/regenerate.sh
```
``````
This has the correct semantic meaning (block of code, not inline code), and also renders appropriately.

Reply:
Done